avro/avro.go

```diff
@@ -0,0 +1,51 @@
+// Package avro provides a user functionality to return the
+// avro encoding of s.
+package avro
+
+import (
+	"fmt"
+
+	"github.com/hamba/avro"
+)
+
+// Marshaller is an interface for marshalling to and from binary using avro
```

Comment:
We should update these comments to refer to "binary encoded byte data" so it's clear what's happening

Reply:
done